mysql-operator/templates/02-rbac.yaml

```diff
@@ -19,11 +19,67 @@ metadata:
   namespace: {{ .Values.operator.namespace}}{{ end }}
 rules:
```

Comment:
We need two ClusterRoles: `mysql-agent` and `mysql-operator` as the components require different permissions (principle of least privilege).

We need a ServiceAccount `mysql-operator` and corresponding ClusterRoleBinding in the namespace into which the operator is being installed.

Users must create a ServiceAccount `mysql-agent` and RoleBinding to the ClusterRole `mysql-agnet` _in each namespace they want to deploy clusters_. We should **not** create this in the default namespace automatically but rather document the requirement clearly.